src/components/ConcurrencyLimitMenu.vue

```diff
@@ -12,27 +12,27 @@
 </template>
 
 <script lang="ts">
-  export default {
-    name: 'ConcurrencyLimitMenu',
+  import { PIconButtonMenu, POverflowMenuItem } from '@prefecthq/prefect-design'
+  import { defineComponent } from 'vue'
+
+  export default defineComponent({
+    name: 'ConcurrencyLimitsMenu',
     expose: [],
     inheritAttrs: false,
-  }
+  })
```

Comment:
I believe this was correct before? We don't usually use define component but rather just export and object.

Reply:
I can make a PR to remove them all

<img width="391" alt="Screenshot 2022-12-09 at 5 12 51 PM" src="https://user-images.githubusercontent.com/40467112/206804119-0bef14ad-2106-4384-82af-512b44d1122f.png">

Reply:
no you're not @pleek91, we moved away from `defineComponent`, must have just missed a few